azerothcore/azerothcore-wotlk
# feat(Core/Config): allow initial (free) amount of Guild Bank tabs

## Changes Proposed:
-  Add a new config in `worldserver.conf` allowing guilds to have an initial (free) amount of guild tabs

## Tests Performed:

Tested in game with a new guild after each `.reload conf`:

- 0 free tabs (default behaviour)
- 1 free tab
- 3 free tabs
- 6 free tabs (see screenshot below)

![image](https://user-images.githubusercontent.com/75517/104792567-12195a00-579f-11eb-8fa0-51dc08733063.png)


## How to Test the Changes:

1) Add/change the `Guild.BankInitialTabs` config in `worldserver.conf` (allowed values 0-6)
2) `.reload conf`
3) Create a new guild `.guild create "Guild Name"`
4) See if the Guild Bank has the correct amount of initial tabs


## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
